Docs/source/theory/cold_fluid_model.rst

```diff
@@ -0,0 +1,135 @@
+.. _theory-cold-fluid-model:
+
+Cold Relativistic Fluid Model
+==========================
+
+An alternate to the representation of the plasma as macroparticles, is the cold relativistic fluid model.
+The cold relativistic fluid model is typically faster to compute than
+particles and useful to replace particles when kinetic effects are negligible. This
+can be done for certain parts of the plasma, such as the background plasma, while still
+representing particle beams as a group of macroparticles. The two models then couple through
+Maxwell's equations.
+
+In the cold limit (zero internal temperature and pressure) of a relativistic plasma, the Maxwell-Fluid
+equations govern the plasma evolution. The fluid equations per species, `s`, are given by,
```

Comment:
double backtick needed in rst to make a verbatim `s`

Reply:
Added and opened a PR: https://github.com/ECP-WarpX/WarpX/pull/4330.